marin/post_training/environments/math_env.py

```diff
@@ -4,15 +4,21 @@
 import datasets
 import jax
 import numpy as np
-from post_training.inference import batch_inference
-from post_training.utils import validate_format
 from tqdm.auto import tqdm
 
+from marin.post_training.inference import batch_inference
```

Comment:
Things in post_training is currently run from post_training as a root folder as opposed to from marin so importing from marin.post_training is not going to work. We will fix it soon by using the executor framework, but for now maybe need to import from post_training instead.